extensions/wikia/ContentReview/ContentReview.hooks.php

```diff
@@ -95,6 +111,49 @@ public static function onRawPageViewBeforeOutput( \RawAction $rawAction, &$text
 		return true;
 	}
 
+	/*
+	 * Adds review status item to top nav tabs in Monobook skin.
+	 * This is an entrypoint for checking review status and submitting changes for review/
+	 * This is attached to the MediaWiki 'SkinTemplateNavigation' hook.
+	 * @param SkinTemplate $skin
+	 * @param array $links Navigation links
+	 * @return bool true
+	 */
+	public static function onSkinTemplateNavigation( $skin, &$links ) {
+		global $wgCityId, $wgTitle;
+
+		if ( $skin->getSkinName() !== 'monobook' ||
+			!self::userCanEditJsPage()
```

Comment:
I always thought our conventions are to put logic operator at the end :)
https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md#conditional-logic
something changed?